js/iframe-coordinator/src/elements/IframeManager.ts

```diff
@@ -0,0 +1,60 @@
+import { HostToClient, validate } from '../messages/HostToClient';
+
+const IFRAME_STYLE =
+  'position: absolute; top: 0; left: 0; width: 100%; height: 100%;';
+
+class IframeManager {
+  private _iframe: HTMLIFrameElement;
+  private _frameLocation: string;
+
+  constructor() {
+    this._frameLocation = 'about:blank';
+
+    this._iframe = document.createElement('iframe');
+    this._iframe.setAttribute('frameborder', '0');
+    this._iframe.setAttribute('style', IFRAME_STYLE);
+    this._iframe.setAttribute('sandbox', 'allow-scripts allow-same-origin');
+  }
+
+  public setFrameLocation(newLocation: string | null) {
```

Comment:
The context here is that it's possible to set the frame location to `null` and get `about:blank`, which is a predictable "no active client" state. We've done that in the past when we've wanted to put an embedded frame in an inactive state without removing it from the DOM. I'm not sure if that use case makes as much sense with this library though.

Reply:
I made a typo in my question.  I mean't can we use newLocation?: string instead.  I am fine with having null be sent, but I don't know why we wouldn't allow undefined as well.

Reply:
Ah, I see. I like the idea of passing `null` because it makes it clear the caller is explicitly and intentionally clearing the location, as opposed to a no-arg call which is a bit more ambiguous, IMO. I'll make the change to allow both though.